src/idiom58ex.fz

```diff
@@ -0,0 +1,5 @@
+ex58 is
+  
+  file := fuzion.std.fileio # shorthand to call the fileio features
+  p := strings.fromBytes(file.read "/home/user/file.xyz") # read takes a relative or an absolute file name
```

Comment:
When you have to put ( / ) around an argument, please separate the argument form the called function by a space. Having no space here means you are using the call syntax using (arg1, args2, arg3), which might be removed.